docs/compilation/composing_functions_with_modules.md

```diff
@@ -249,9 +249,9 @@ class Collatz:
         return ans, is_one
```

Comment:
Mention the automatic wire tracing and give a link as a point number 4 (l 233)

Reply:
Will do !